openstack_plugin_common/__init__.py

```diff
@@ -1154,8 +1154,7 @@ def handle_no_resource_id_operations(_ctx, operation_name, instance_id,
             del _ctx.instance.runtime_properties[runtime_prop]
     # skip operations since resource_id is not assigned to take action
     elif operation_name in CLOUDIFY_OPERATIONS_WITHOUT_CREATE:
-        _ctx.logger.info("ignoring action since resource_id is not set")
-        return False
+        raise RecoverableError("Waiting for resource_id to be available")
```

Comment:
@ahmadiesa-abu Is the purpose of this execption just to retry ? can we use OperationRetry

Reply:
well yes , it is the same idea , but why error because it was not suppose to reach this far [ because you can't do any operation on OpenStack resource with external_id missing ]

Reply:
What do you think about ignore uninstall action(stop,delete) without resource_id(as we have before) and retry only for install actions(establish, configure, start).

Reply:
@0lvin yes, you are right and I did the change to handle the cases